usr/share/rear/lib/global-functions.sh

```diff
@@ -837,23 +840,60 @@ function umount_mountpoint() {
 
     ### otherwise, try to kill all processes that opened files on the mount.
     # TODO: actually implement this
+    sleep 1
+    # We could use fuser to kill the PID holding the mount?
+    # That would mean that the 'fuser' command would become required, no?
+    # $ fuser -v -M -m /mnt
+    #                 USER        PID ACCESS COMMAND
+    #/mnt:                root     kernel mount /mnt
+    #                 gdhaese1  3408944 ..c.. bash
+    ## The line containing "kernel mount" we may not kill, only processes that occupy the mount point.
+    ## In above example that would be PID 3408944
+    PIDs_2_kill="$(fuser -v -M -m $mountpoint 2>&1 | grep -v -E '(PID|kernel)' | awk '{print $2}')"
+    if [[ -n "$PIDs_2_kill" ]] ; then
+	Log "Kill processes that prevent the umount of $mountpoint - PIDs $PIDs_2_kill"
+        kill -USR1 "$PIDs_2_kill"
```

Comment:
I wonder if it is reasonably safe to "just terminate" a process
(via SIGUSR1 in this case - why not SIGTERM as usual?)
unconditionally without knowing what process it is
and without giving the user a chance to intervene
(perhaps that process is critical for the user).